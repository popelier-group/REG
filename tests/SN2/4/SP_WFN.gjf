# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -0.43210700   -0.00008400    0.00022300
 H                 -0.73753300   -0.87982400   -0.54754700
 H                 -0.73761300    0.91416300   -0.48772700
 H                 -0.73738400   -0.03462800    1.03603000
 Br                 1.60582200   -0.00000100    0.00001000
 Cl                -3.41199900    0.00004000   -0.00012000

frame.wfn
