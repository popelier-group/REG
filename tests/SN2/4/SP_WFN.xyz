
6

C       -0.321557    0.004651    0.037947
H       -0.529551   -0.540299    0.947369
H       -0.738562   -0.477600   -0.834302
H       -0.606391    1.044484    0.107945
Br        1.702186   -0.023469   -0.200437
Cl       -3.280745    0.045113    0.386271
