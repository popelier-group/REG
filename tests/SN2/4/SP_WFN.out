 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.4/Gau-70827.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.4/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe PID=     70830.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -0.43211  -0.00008   0.00022 
 H                    -0.73753  -0.87982  -0.54755 
 H                    -0.73761   0.91416  -0.48773 
 H                    -0.73738  -0.03463   1.03603 
 Br                    1.60582  0.         0.00001 
 Cl                   -3.412     0.00004  -0.00012 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -0.432107   -0.000084    0.000223
      2          1           0       -0.737533   -0.879824   -0.547547
      3          1           0       -0.737613    0.914163   -0.487727
      4          1           0       -0.737384   -0.034628    1.036030
      5         35           0        1.605822   -0.000001    0.000010
      6         17           0       -3.411999    0.000040   -0.000120
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.080407   0.000000
     3  H    1.080406   1.794984   0.000000
     4  H    1.080409   1.795013   1.795004   0.000000
     5  Br   2.037929   2.562268   2.562278   2.562255   0.000000
     6  Cl   2.979892   2.868206   2.868052   2.868514   5.017821
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -0.321557    0.004651    0.037947
      2          1           0       -0.529551   -0.540299    0.947369
      3          1           0       -0.738562   -0.477600   -0.834302
      4          1           0       -0.606391    1.044484    0.107945
      5         35           0        1.702186   -0.023469   -0.200437
      6         17           0       -3.280745    0.045113    0.386271
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         155.6333908           0.8202729           0.8202728
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       176.1866084329 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  9.32D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147525.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 EnCoef did   100 forward-backward iterations
 SCF Done:  E(RB3LYP) =  -3074.39457467     A.U. after   12 cycles
            NFock= 12  Conv=0.12D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.68683-101.25816 -62.33986 -56.15969 -56.15653
 Alpha  occ. eigenvalues --  -56.15653 -10.07957  -9.17127  -8.54923  -6.92995
 Alpha  occ. eigenvalues --   -6.92936  -6.92936  -6.37894  -6.36852  -6.36852
 Alpha  occ. eigenvalues --   -2.49010  -2.48715  -2.48715  -2.47889  -2.47889
 Alpha  occ. eigenvalues --   -0.65983  -0.53880  -0.50972  -0.30032  -0.30032
 Alpha  occ. eigenvalues --   -0.24086  -0.15527  -0.15527  -0.05905  -0.05788
 Alpha  occ. eigenvalues --   -0.05788
 Alpha virt. eigenvalues --    0.09872   0.10978   0.14089   0.14089   0.16188
 Alpha virt. eigenvalues --    0.17433   0.18057   0.18058   0.21498   0.21931
 Alpha virt. eigenvalues --    0.21932   0.24324   0.24324   0.24631   0.31048
 Alpha virt. eigenvalues --    0.31049   0.31643   0.34176   0.40589   0.50266
 Alpha virt. eigenvalues --    0.52347   0.52347   0.54085   0.54086   0.66924
 Alpha virt. eigenvalues --    0.67723   0.67723   0.69437   0.74820   0.74821
 Alpha virt. eigenvalues --    0.75808   0.77355   0.77355   0.82776   0.82776
 Alpha virt. eigenvalues --    0.84338   0.95474   1.08354   1.14775   1.14775
 Alpha virt. eigenvalues --    1.15232   1.15233   1.16401   1.23188   1.23189
 Alpha virt. eigenvalues --    1.25050   1.56608   1.67510   1.67511   1.80990
 Alpha virt. eigenvalues --    1.84808   1.84808   2.08278   2.12612   2.12613
 Alpha virt. eigenvalues --    2.29066   2.30999   2.31000   2.52134   2.52135
 Alpha virt. eigenvalues --    2.56150   2.60277   2.64573   2.64574   2.68538
 Alpha virt. eigenvalues --    2.68539   3.12598   3.14446   3.85873   3.85874
 Alpha virt. eigenvalues --    4.27988   4.27988   4.37162   4.37163   4.65876
 Alpha virt. eigenvalues --    6.73536   7.69391   7.69392   7.79672   9.87716
 Alpha virt. eigenvalues --   24.04371  26.08063  26.08066  26.20330  48.02873
 Alpha virt. eigenvalues --  215.84281 289.89381 289.89381 290.014611020.80674
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.064983   0.411323   0.411335   0.411297   0.096521  -0.055413
     2  H    0.411323   0.528997  -0.035825  -0.035810  -0.055425   0.007621
     3  H    0.411335  -0.035825   0.528973  -0.035815  -0.055420   0.007621
     4  H    0.411297  -0.035810  -0.035815   0.529047  -0.055431   0.007622
     5  Br   0.096521  -0.055425  -0.055420  -0.055431  35.370394   0.007954
     6  Cl  -0.055413   0.007621   0.007621   0.007622   0.007954  17.913296
 Mulliken charges:
               1
     1  C   -0.340046
     2  H    0.179117
     3  H    0.179130
     4  H    0.179091
     5  Br  -0.308592
     6  Cl  -0.888700
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C    0.197292
     5  Br  -0.308592
     6  Cl  -0.888700
 Electronic spatial extent (au):  <R**2>=           1185.2917
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=             10.6364    Y=             -0.1459    Z=             -1.2522  Tot=             10.7109
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -96.5653   YY=            -43.8035   ZZ=            -44.5252
   XY=              0.7257   XZ=              6.2133   YZ=             -0.0854
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -34.9340   YY=             17.8278   ZZ=             17.1062
   XY=              0.7257   XZ=              6.2133   YZ=             -0.0854
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=            227.4476  YYY=             -0.0764  ZZZ=             -8.4576  XYY=             23.0209
  XXY=             -2.5092  XXZ=            -21.3757  XZZ=             25.1440  YZZ=             -1.2072
  YYZ=             -2.6182  XYZ=              0.1567
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=          -2015.2103 YYYY=            -70.3260 ZZZZ=            -90.4139 XXXY=             16.7463
 XXXZ=            143.1719 YYYX=              9.5309 YYYZ=             -1.1198 ZZZX=             86.6979
 ZZZY=             -1.3698 XXYY=           -266.5913 XXZZ=           -276.5184 YYZZ=            -26.7785
 XXYZ=             -1.0663 YYXZ=             28.5782 ZZXY=              3.9232
 N-N= 1.761866084329D+02 E-N=-7.703731689869D+03  KE= 3.070003254641D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE333\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\ROOT\11-Nov-20
 20\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\C,0,
 -0.432107,-0.000084,0.000223\H,0,-0.737533,-0.879824,-0.547547\H,0,-0.
 737613,0.914163,-0.487727\H,0,-0.737384,-0.034628,1.03603\Br,0,1.60582
 2,-0.000001,0.00001\Cl,0,-3.411999,0.00004,-0.00012\\Version=ES64L-G16
 RevC.01\State=1-A\HF=-3074.3945747\RMSD=1.191e-09\Dipole=4.213991,-0.0
 001024,0.0002916\Quadrupole=-26.5238485,13.2619118,13.2619367,0.000501
 3,-0.0015164,0.0000035\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


 KNOWLEDGE IS OF TWO KINDS:  WE KNOW A SUBJECT OURSELVES OR WE KNOW 
 WHERE WE CAN FIND INFORMATION UPON IT.

                                    -- SAMUEL JOHNSON
 Job cpu time:       0 days  0 hours  0 minutes 29.5 seconds.
 Elapsed time:       0 days  0 hours  0 minutes 15.4 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 18:34:08 2020.
